book/src/faq.md

```diff
@@ -1,5 +1,18 @@
 # FAQ
 
+## Why aren't property values updating for my service that doesn't notify changes?
+
+A common issue might arise when using zbus and your service's property values aren't updating. 
+This is due to zbus' default caching policy, which updates the value of a property only when 
```

Comment:
This is specific to `Proxy` but the text makes it sound like it's the general behavior (e.g you won't face this with the low-level `MessageStream` API). Would be good to be specific here.